gulpfile.js

```diff
@@ -412,9 +419,8 @@ gulp.task('js', function() {
 });
 
 
-function collector(file, memo) {
-  return Object.assign({}, JSON.parse(file.contents), memo);
-}
+const collector = (file, memo) =>
+  Object.assign(memo, JSON.parse(file.contents));
```

Comment:
Swapping the assignment round causes no ill effects? Do they share no common prop names?

Reply:
No, they should always be unique